lib/lightning/application.ex

```diff
@@ -9,7 +9,7 @@ defmodule Lightning.Application do
   @impl true
   def start(_type, _args) do
     # mnesia startup
-    :mnesia.stop()
+    # :mnesia.stop()
```

Comment:
@taylordowns2000 This is the recommended pattern for mnesia, while not compulsory in our current application shape - in the event of other applications that are started before or after this one that also use mnesia, you must always stop mnesia before creating a schema. I think we should leave this here.

Reply:
@elias-ba please change this back.

Reply:
Done